botocore/utils.py

```diff
@@ -320,15 +319,20 @@ def _get_request(self, url, num_attempts=1, needs_retry=None):
                     "service request to %s: %s", url, e, exc_info=True)
         raise _RetriesExceededError()
 
-    def _is_non_ok_response(self, response):
-        return response.status_code != 200
+    def _does_not_contain_role_name(self, response):
+        return (
+            self._is_non_ok_response(response) or
+            self._is_empty(response)
+        )
 
     def _does_not_contain_credentials(self, response):
-        if (self._is_non_ok_response(response) or
+        return (self._is_non_ok_response(response) or
```

Comment:
Can we make the formatting on this consistent with the other function like this:
```python
return (
    self._is_...
)
```